charts/fluid/v0.7.0/templates/csi/daemonset.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       labels:
         app: csi-nodeplugin-fluid
     spec:
-      serviceAccount: fluid-csi
+      serviceAccountName: fluid-csi
```

Comment:
Please don't update the released version. thanks.

Reply:
done